### ISSUE TITLE
ERROR: Cannot find module './getHash'

### ISSUE THREAD

**Issue by fliaping** [minwe/qn-store#11]
upgrade to ghost 0.11.1 and occur error : Cannot find module './getHash'

but when I use local storage , it goes well

`
root@xp:/home/www/ghost# cnpm start --production

> ghost@0.11.1 start /home/www/ghost
> node index

ERROR: Cannot find module './getHash'

 Error
    at Error.IncorrectUsage (/home/www/ghost/core/server/errors/incorrect-usage.js:3:18)
    at Object.getStorage (/home/www/ghost/core/server/storage/index.js:36:19)
    at setupMiddleware (/home/www/ghost/core/server/middleware/index.js:126:44)
    at /home/www/ghost/core/server/index.js:188:9
    at tryCatcher (/home/www/ghost/node_modules/.3.4.6@bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/home/www/ghost/node_modules/.3.4.6@bluebird/js/release/promise.js:510:31)
    at Promise._settlePromise (/home/www/ghost/node_modules/.3.4.6@bluebird/js/release/promise.js:567:18)
    at Promise._settlePromise0 (/home/www/ghost/node_modules/.3.4.6@bluebird/js/release/promise.js:612:10)
    at Promise._settlePromises (/home/www/ghost/node_modules/.3.4.6@bluebird/js/release/promise.js:691:18)
    at Promise._fulfill (/home/www/ghost/node_modules/.3.4.6@bluebird/js/release/promise.js:636:18)
    at PromiseArray._resolve (/home/www/ghost/node_modules/.3.4.6@bluebird/js/release/promise_array.js:125:19)
    at PromiseArray._promiseFulfilled (/home/www/ghost/node_modules/.3.4.6@bluebird/js/release/promise_array.js:143:14)
    at Promise._settlePromise (/home/www/ghost/node_modules/.3.4.6@bluebird/js/release/promise.js:572:26)
    at Promise._settlePromise0 (/home/www/ghost/node_modules/.3.4.6@bluebird/js/release/promise.js:612:10)
    at Promise._settlePromises (/home/www/ghost/node_modules/.3.4.6@bluebird/js/release/promise.js:691:18)
    at Async._drainQueue (/home/www/ghost/node_modules/.3.4.6@bluebird/js/release/async.js:138:16)
    at Async._drainQueues (/home/www/ghost/node_modules/.3.4.6@bluebird/js/release/async.js:148:10)
    at Immediate.Async.drainQueues [as _onImmediate](/home/www/ghost/node_modules/.3.4.6@bluebird/js/release/async.js:17:14)
    at processImmediate [as _immediateCallback](timers.js:383:17)

`

**Comment by minwe** [comment]
Fixed in 1.2.1.

**Comment by fliaping** [comment]
thank you, but I find another problem, when I set `hashAsBasename: false` , it will happen ERROR: name is not defined. if hashAsBasename as true, it's ok

```
ERROR: name is not defined

 ReferenceError: name is not defined
    at QiniuStore.getFileKey (/home/www/ghost/content/storage/qn-store/index.js:87:35)
    at /home/www/ghost/content/storage/qn-store/index.js:32:11
    at Promise._execute (/home/www/ghost/node_modules/.3.4.6@bluebird/js/release/debuggability.js:299:9)
    at Promise._resolveFromExecutor (/home/www/ghost/node_modules/.3.4.6@bluebird/js/release/promise.js:481:18)
    at new Promise (/home/www/ghost/node_modules/.3.4.6@bluebird/js/release/promise.js:77:14)
    at QiniuStore.save (/home/www/ghost/content/storage/qn-store/index.js:31:10)
    at /home/www/ghost/core/server/api/upload.js:24:22
    at tryCatcher (/home/www/ghost/node_modules/.3.4.6@bluebird/js/release/util.js:16:23)
    at /home/www/ghost/node_modules/.3.4.6@bluebird/js/release/method.js:15:34
    at apiHandler (/home/www/ghost/core/server/api/index.js:242:16)
    at Layer.handle [as handle_request] (/home/www/ghost/node_modules/.4.14.0@express/lib/router/layer.js:95:5)
    at next (/home/www/ghost/node_modules/.4.14.0@express/lib/router/route.js:131:13)
    at /home/www/ghost/core/server/middleware/validation/upload.js:28:9
    at Layer.handle [as handle_request] (/home/www/ghost/node_modules/.4.14.0@express/lib/router/layer.js:95:5)
    at next (/home/www/ghost/node_modules/.4.14.0@express/lib/router/route.js:131:13)
    at Immediate.<anonymous> (/home/www/ghost/node_modules/.1.2.0@multer/lib/make-middleware.js:52:37)
    at Immediate.immediate._onImmediate (timers.js:440:18)
    at processImmediate [as _immediateCallback] (timers.js:383:17)
```

**Comment by minwe** [comment]
Sorry for my mistake, will be fixed tonight.

**Comment by minwe** [comment]
@paynexu Fixed in 1.2.2.

**Comment by fliaping** [comment]
thank you
